### PR TITLE
Add requests_mock

### PR DESCRIPTION
**Pull request by AbdealiJK**

### CODE REVIEW

File: recipes/requests_mock/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  url: https://pypi.io/packages/source/r/requests_mock/requests-mock-{{ version }}.tar.gz
+  sha256: bd86970d6c52cc97071f5185aa594de6a997a5ca63b3bb36aceb9bb9db49294b
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - setuptools
```
Comment (synapticarbors) [inline recipes/requests_mock/meta.yaml]: please add `python` as a `build` requirement

File: recipes/requests_mock/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    - pbr
+
+  run:
```
Comment (synapticarbors) [inline recipes/requests_mock/meta.yaml]: please add `python` as a `run` requirement

File: recipes/requests_mock/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    - requests >=1.1
+    - six
+    - fixtures
+
+test:
+  imports:
+    - "{{ name }}"
```
Comment (synapticarbors) [inline recipes/requests_mock/meta.yaml]: Please don't use jinja template variables here. Just list the explicit module

File: recipes/requests_mock/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+about:
+  home: https://github.com/openstack/requests-mock
+  license: Apache 2.0
```
Comment (synapticarbors) [inline recipes/requests_mock/meta.yaml]: add `license_file: LICENSE`

File: recipes/requests_mock/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/r/requests_mock/requests-mock-{{ version }}.tar.gz
+  sha256: bd86970d6c52cc97071f5185aa594de6a997a5ca63b3bb36aceb9bb9db49294b
+
+build:
```
Comment (synapticarbors) [inline recipes/requests_mock/meta.yaml]: This recipe might be a candidate for `noarch: python`:

https://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

**Comment by sodre** [comment]
I am willing to finish the changes and be a co-maintainer with @AbdealiJK

**Pull request closed without merging**